resources/views/errors/status.blade.php

```diff
@@ -0,0 +1,11 @@
+@if (Session::has('status'))
+    <div>{{ Session::get('status') }}</div>
+@endif
+
+@if(isset($errors) && $errors->any())
```

Comment:
@DFurnes without this, I was getting `Undefined variable: errors` when I visited the homepage cause it wasn't defined inside of the middleware. I can move it so we don't need this anymore